src/components/pages/RoleSignup/Applications/AppSuccess.js

```diff
@@ -1,28 +1,47 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Col, Row, Typography, Button } from 'antd';
+import { connect } from 'react-redux';
+import { getProfile } from '../../../../state/actions/userProfile/getProfile';
+import axiosWithAuth from '../../../../utils/axiosWithAuth';
 import { Link } from 'react-router-dom';
 const { Title } = Typography;
 
-const AppSuccess = () => {
+const AppSuccess = ({ isAuthenticated, getProfile }) => {
+  const [user, setUser] = useState({});
+  useEffect(() => {
+    axiosWithAuth()
+      .get(`/profile/current_user_profile/`)
+      .then(user => {
+        setUser(user.data);
+        getProfile(user.data.profile_id);
+      });
+  }, [isAuthenticated]);
+
   return (
     <>
       <Row align="center" gutter={[16, 16]} style={{ marginTop: '5vh' }}>
         <Col span={24} align="middle">
-          <Title level={2}>Thank you!</Title>
+          <Title level={2}>Thank you {user.first_name}!</Title>
         </Col>
         <Col span={24} align="middle">
           <Typography>
-            Your application was successfully submitted! Someone at Underdog
-            Devs will contact you shortly in regards to your application
-            approval.
+            Your application has been received, keep an eye on your email, as
+            we'll email you when your application has been approved.
```

Comment:
I would change the wording, as this implies that the applicant will get an email of their approval within a time frame, rather than an email regarding if their application will be approved or not.

Reply:
Did you mean the text in green implies a time frame? One of the reasons we changed it was to make it less time-bound than the red text. Just making sure we're talking about the same lines.

Reply:
Yeah, I'm referring to the green line. It just refers that the applicant will receive an email when they're approved(unless I'm mistaken and Underdog Devs approves all applicants), not if they'll be approved or not.

Reply:
Thanks for catching that. We just changed **when** to **if**.
let us know if you think anything else should be changed for that.